### ISSUE TITLE
Activation of `global install` bins are overwriting the path, affect program like `pueue add`

### ISSUE THREAD

**Issue by way-zer** [prefix-dev/pixi#2507]
### Checks

- [x] I have checked that this issue has not already been reported.

- [x] I have confirmed this bug exists on the [latest version](https://github.com/prefix-dev/pixi/releases) of pixi, using `pixi --version`.

### Reproducible example

```shell
pixi global install "pueue" -c dnachun -c conda-forge
pueued -d #start deamon
export PATH="TEST:$PATH" # or pixi shell in another project
pueue add env 
pueue log -f 1 | grep PATH # will get wrong PATH
```

### Issue description

May like #133. `pixi global` will overwrite PATH and CONDA_PREFIX(no affect this example), so the actual execuable will get wrong PATH variable.

Maybe provide a way to expose binary without activation, and maybe use symbol link.

### Expected behavior

not overwrite or just append necessary paths to PATH variable.

**Comment by Hofer-Julian** [comment]
Thanks for the issue. We'd be open to a --no-activation option. Just haven't gotten around implementing it yet

**Comment by wolfv** [comment]
@Hofer-Julian  didn't we discuss to keep the path dynamic? We could easily react to PATH changes without changing the rest of the logic

**Comment by Hofer-Julian** [comment]
> [@Hofer-Julian](https://github.com/Hofer-Julian) didn't we discuss to keep the path dynamic? We could easily react to PATH changes without changing the rest of the logic

Yes, that is indeed the more elegant solution for this specific problem

**Comment by way-zer** [comment]
It seems break many things, like `pixi global make`. When use `make`, it can't found binary in current PATH.

**Comment by way-zer** [comment]
Seems fixed in v0.38.0
